uber/templates/macros.html

```diff
@@ -0,0 +1,115 @@
+{% macro ie7_compatibility_check() -%}
+    <!--[if lte IE 7]>
+        <script>
+            if (!document.documentMode) { //if documentMode exists, this is a later IE in compatibility mode
+                alert("This website requires IE 8 or later.");
+            }
+        </script>
+    <![endif]-->
+{%- endmacro %}
+
+
+{% macro stripe_button(label) -%}
+    <button class="stripe-button-el">
+        <span class="display: block; min-height: 30px;">{{label}}</span>
+    </button>
+{%- endmacro %}
+
+
+{% macro popup_link(href, text='<sup>?</sup>') -%}
+    <a onClick="window.open('{{ href }}', 'info', 'toolbar=no,height=500,width=375,scrollbars=yes').focus(); return false;"
+       href="{{ href }}">{{ text|safe }}</a>
+
+    {#
+    TODO: integrate these changes from upstream, if needed. they may not be needed anymore
+
+    inner_text = "window.open('{self.href}', 'info', 'toolbar=no,height=500,width=375,scrollbars=yes').focus(); return false;".format(self=self)
+    inner_text = inner_text.replace("'", "&quot;")
+    return "<a onClick='{inner_text}' href='{self.href}'>{self.text}</a>".format(inner_text=inner_text, self=self)
+    #}
```

Comment:
we should probably keep or kill these changes inside popup_link.   probably something changed in either master or in upstream rams during the timeframe I originally had this PR open, resulting in changes that were not easy to merge.